rebar.config.script

```diff
@@ -186,18 +186,13 @@ MakeDep = fun
         {AppName, ".*", {git, Url, Version}, Options}
 end.
 
-ErlOpts = case os:getenv("ERL_OPTS") of
-    false -> [];
-    Opts -> [list_to_atom(O) || O <- string:tokens(Opts, ",")]
-end.
-
 AddConfig = [
     {require_otp_vsn, "21|22|23|24"},
     {deps_dir, "src"},
     {deps, lists:map(MakeDep, DepDescs ++ OptionalDeps)},
     {sub_dirs, SubDirs},
     {lib_dirs, ["src"]},
-    {erl_opts, [{i, "../"} | ErlOpts]},
+    {erl_opts, [{i, "../"} | compile:env_compiler_options()]},
```

Comment:
Since compiler consults those automatically, we probably don't have to set it at all?